Earthfile

```diff
@@ -182,10 +182,12 @@ release-archive:
     WORKDIR /s
     COPY --dir .git .
     COPY (+sbom-download/augmented-sbom.json --branch=$sbom_branch) cyclonedx.sbom.json
+    COPY etc/ssdlc.md ssldc_compliance_report.md
```

Comment:
Any reason not to simply name `etc/ssdlc.md` and leave it at its committed location? The copy into root directory seems superfluous (the files being in the root directory isn't a requirement AFAIK). Having two copies of the report in a release archive may be somewhat confusing.

Reply:
I'll just rename the file and allow `git archive` to grab it.